src/Education/Education.csproj

```diff
@@ -29,6 +29,14 @@
     </Page>
   </ItemGroup>
 
+  <ItemGroup Label="Images">
+    <None Remove="XamlGenerator\images\severity\*.png" />
+    <Resource Include="XamlGenerator\images\severity\*.png" />
+
+    <None Remove="XamlGenerator\images\type\*.png" />
+    <Resource Include="XamlGenerator\images\type\*.png" />
+  </ItemGroup>
+
```

Comment:
I copied all of the relevant images from SLI. There are multiple versions of each each in different sizes; currently we only use one size of each.